CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
```

Comment:
3.0.0 is published (whoops) on 3/29/2021. This'll need to be a new entry for 3.1.0.

Reply:
And we can bump the number in package.json too.

Reply:
Looks like I forgot to add the dates to this one so I'll toss those in there while I'm in here

Reply:
I think you may have branched off of an older copy of the repo / dev branch since the dates were in there already in dev.